packages/teams-js/src/private/otherAppStateChange.ts

```diff
@@ -110,6 +113,32 @@ export function unregisterAppInstallationHandler(): void {
   removeHandler(ApiName.OtherAppStateChange_Install);
 }
 
+/**
+   * @hidden
+   * @beta
+   * @internal
+   * Limited to Microsoft-internal use
+   *
+   * This function should be called by the Store App to notify the host that the
+   * app with the given appId has been installed.
+   *
+   * @throws Error if {@link app.initialize} has not successfully completed or if the platform
+   * does not support the otherAppStateChange capability.
+   */
+  export function notifyAppInstall(appId: AppId): void {
```

Comment:
Please add unit tests for this new function

Reply:
added